React/Views/RCTPicker.m

```diff
@@ -12,6 +12,9 @@
 #import "RCTUtils.h"
 
 @interface RCTPicker() <UIPickerViewDataSource, UIPickerViewDelegate>
+
+ #define UIColorFromRGB(rgbValue) [UIColor colorWithRed:((float)((rgbValue & 0xFF0000) >> 16))/255.0 green:((float)((rgbValue & 0xFF00) >> 8))/255.0 blue:((float)(rgbValue & 0xFF))/255.0 alpha:1.0]
```

Comment:
We have the `[RCTConvert UIColor:json]` method for this already. Import "RCTConvert.h" to use it.